typescript-test-samples/lambda-handler-dynamodb/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "lambda-handler-dynamodb",
+  "version": "0.1.0",
+  "bin": {
+    "lambda-handler-dynamodb": "bin/lambda-handler-dynamodb.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk",
+    "test:unit": "jest --selectProjects unit",
```

Comment:
`test:unit` and `test:integration` are not configured for this project, we should drop them.